pywwt/layers.py

```diff
@@ -302,7 +303,71 @@ def add_image_layer(
         # has previously been processed. As usual, the logic is a bit
         # complicated since we aim to allow the user to give us just about
         # anything as an input.
+        if any(isinstance(image, fitsy_type) for fitsy_type in [
+            astropy.io.fits.HDUList,
+            astropy.io.fits.ImageHDU,
+            astropy.io.fits.PrimaryHDU
+        ]):
```

Comment:
Oh, not worth editing things, but IIRC `isinstance(image, (HDUList, ImageHDU, PrimaryHDU))` actually has these exact semantics.

Reply:
Ah, yep, I forgot that `isinstance` can take a tuple